### ISSUE TITLE
Does the OHIF Viewer support segmentation when Google Cloud Healthcare is set as its data source?

### ISSUE THREAD

**Issue by omartoutounji** [OHIF/Viewers#817]
I noticed segmentation doesn't really work when gcloud is set as its data source. What do I mean by that?

The OHIF viewer displays segmentation successfully when orthanc is set as its data source whereas it doesn't when gcloud is its data source. Keep in mind that I am using the exact same DICOM data with both data sources.

So does the OHIF Viewer support segmentation when Google Cloud Healthcare is set as its data source? I'm not even sure wether this is a gcloud or an OHIF issue. If it's an OHIF issue, would really appreciate some guidance to draft a PR.

**Comment by swederik** [comment]
This is because Google Cloud doesn't support WADO-URI and right now the Cornerstone extension is using WADO-URI as a hacky way to get the SEG. 

I had a commit where I removed this but I think I didn't get around to merging it before the monorepo changes.

TLDR: Known bug with a known solution needs to be fixed in the short term.

**Comment by dannyrb** [comment]
Scoped here: Fix data retrieval issues for local file and Google Cloud use cases (i.e. avoid fetch and WADO-URI)
